docs/README.md

```diff
@@ -525,6 +525,12 @@ successText = "Thanks!"
 
 ?> This parameter is optional. Default value is empty. Possible values are any defined function.
 
+#### *callback* arguments
+
+- hash: *string* - the hash of the broadcasted transaction
+- amount: *string* - how amount money is being requested
+- paymentId: *string* - the payment unique identifier
+
```

Comment:
After actually looking at the data payload coming from the server and thinking about this for a bit, I've had a change of opinion on this.

Why not just have only the tx object be available here and they can dig into it and use whatever they need from that?

This way, we wouldn't have to go back in and add/remove/update the arguments when they change, right?

Reply:
I made this for the server as we're sending a lot of useless data to the client: https://github.com/PayButton/paybutton-server/issues/811 -- maybe we can think about updating the docs to include all of the tx data once that's eventually done? I'm not really concerned about having all of this in the docs beforehand since we made decide to keep/remove other stuff in the payload that I'm not thinking about.

Reply:
An argument against my idea here of just passing `tx` for them to use is that this is really only applicable to `on-success` and `on-transaction` but not `on-open` and `on-close`.

Thoughts?

Reply:
Estevao is good with this change so let's go ahead with it in this branch and update the docs again.